apps/expo/src/app/calendar.tsx

```diff
@@ -1,13 +1,82 @@
 import React from "react";
-import { Text } from "react-native";
+import { Image, Text, View } from "react-native";
 import { SafeAreaView } from "react-native-safe-area-context";
 import { Stack } from "expo-router";
+import { useUser } from "@clerk/clerk-expo";
+
+import EventTab from "~/components/EventTab";
+import Logo from "../../assets/logo.png";
```

Comment:
try doing `import * as Logo from "../../assets/logo.png"` -- I think this should fix the lint error hopefully?